src/python/pants/engine/internals/graph.py

```diff
@@ -735,7 +736,13 @@ def create_live_and_deleted_gets(
             sources_set = deleted_files
 
         build_file_addresses = await MultiGet(
-            Get(BuildFileAddress, Address, tgt.address) for tgt in candidate_tgts
+            Get(
+                BuildFileAddress,
+                BuildFileAddressRequest(
+                    tgt.address, description_of_origin="<owners rule - cannot trigger>"
```

Comment:
My thinking is that we use Specs globs above to discover what targets are valid. So our universe of targets in-play must be valid, by definition.

Lmk if it would help to comment that